birdhouse/components/weaver/config/weaver/data_sources.json.template

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
So weaver can handle .yml config file.  This seems to be the only .json config file.  Any plan later to make this .json a .yml as well so we can add comment to the config file for inplace clarification?

Reply:
Yes. Easy to fix. Just need `yaml.safe_load` instead of `json.load` as it handles both YAML/JSON.

Reply:
https://github.com/crim-ca/weaver/pull/206

Reply:
Yes